### PR TITLE
fix: sanitize tokens for the blacklist

### PR DESCRIPTION
**Pull request by abc3**

### CODE REVIEW

File: lib/supavisor_web/router.ex [hunk 1]
```diff
@@ -89,6 +89,7 @@ defmodule SupavisorWeb.Router do
     blocklist = Application.fetch_env!(:supavisor, blocklist_key)
 
     with ["Bearer " <> token] <- get_req_header(conn, "authorization"),
+         token <- Regex.replace(~r/\s|\n/, URI.decode(token), ""),
```
Comment (filipecabaco) [inline lib/supavisor_web/router.ex:92]: why not 
```
blocklist =
      Application.get_env(:realtime, blocklist_key, [])
      |> Enum.map(&String.trim/1)
      |> Enum.map(&String.downcase/1)

```

Comment (abc3) [reply lib/supavisor_web/router.ex:92]: I repeated the same approach as in JwtAuthToken to ensure we manipulate the same string

Comment (filipecabaco) [reply lib/supavisor_web/router.ex:92]: but why URI decode it? really curious about it.

Comment (abc3) [reply lib/supavisor_web/router.ex:92]: you can hide some symbols via HTML characters, which token sanitization can skip but the JWT library can decode

Comment (filipecabaco) [reply lib/supavisor_web/router.ex:92]: TIL 👀 will copy pasta it into realtime 🍝

Comment (filipecabaco) [reply lib/supavisor_web/router.ex:92]: should we do the same to the blocklist tokens?

Comment (abc3) [reply lib/supavisor_web/router.ex:92]: I guess not, because we can check that they are without spaces or new lines?

Comment (abc3) [reply lib/supavisor_web/router.ex:92]: wdyt?

Comment (filipecabaco) [reply lib/supavisor_web/router.ex:92]: fair... fair...

**Pull request merged**